deepchem/utils/equivariance_utils.py

```diff
@@ -0,0 +1,174 @@
+import math
+import torch
+from typing import Optional
+
+
+def su2_generators(j: int) -> torch.Tensor:
+    """Get the generators of the SU(2) Lie algebra for a given quantum angular momentum.
+
+    Parameters
+    ----------
+        j (int): The quantum angular momentum (spin) value.
+
+    Returns
+    -------
+        torch.Tensor: A stack of three SU(2) generators, corresponding to J_x, J_z, and J_y.
```

Comment:
Can you add references? Check https://numpydoc.readthedocs.io/en/latest/format.html for style for references